api/sixtyfps-node/native/lib.rs

```diff
@@ -276,6 +276,15 @@ fn to_js_value<'cx>(
             &format!("#{:02x}{:02x}{:02x}{:02x}", c.red(), c.green(), c.blue(), c.alpha()),
         )
         .as_value(cx),
+        Value::Model(model) => {
```

Comment:
That's a good point, I agree. Will look into that.

I think `JsModel` is used & created when assigning a "model like" object from JS to a model property in .60. So in a sense most of our tests (like `write_to_model.60`) that create a `sixtyfps::ArrayModel` in JS and then assign it to a model property cover that.

> There is also an occurence in api/sixtyfps-node/native/lib.rs:180 that needs to be changed to use a Model

Indeed. I wonder why this didn't break anything. I guess because we continue to support that within the interpreter, i.e. `ValueModel` in the interpreter supports `Value::Array`.

Reply:
> We should try to do a as_dyn of a JsModel and then return the inner object.

I've made a commit that tests and fixes the round-tripping.